lessonbuilder/components/src/java/org/sakaiproject/lessonbuildertool/util/LessonsSubNavBuilder.java

```diff
@@ -50,12 +50,14 @@ public class LessonsSubNavBuilder {
     private String siteId;
     private boolean isInstructor;
     private Map<String, ArrayList<Map<String, String>>> subnavData;
+    private ArrayList<Map<String, String>> topLevelPageProps;
```

Comment:
I chose ArrayList instead of Collection because sequence matters with respect to determining which pages are prerequisites of which. The type for topLevelPageProps mirrors the nested ArrayList type for subnavData, where the sequence of the data also matters.

Reply:
Collection is a super type so it's fine!

plus you wouldn't normally use the implementing class ArrayList but rather use it's interface List.